text/0000-partial-lambda-args.md

```diff
@@ -0,0 +1,184 @@
+- Feature Name: `partial_lambda_args`
+- Start Date: 2020-07-14
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Pattern matching has this nice mechanism to be _partial_: you can pattern match what you are
+interested in, and then use the `..` syntax to _ignore everything else_. Example:
+
+```rust
+let (a, b, ..) = (1, true, "Hello, world!"); // here
+
+struct Foo {
+  x: u32,
```

Comment:
Idiomatic Rust uses 4 spaces. It's worth running `rustfmt` on your code.

Reply:
How is that any relevant to the RFC?

Reply:
I'm not @shepmaster, but I know from my perspective having a consistent style across RFCs is one less distraction from the content of the RFC itself. Does that make sense?